L/licensecheck/build_tarballs.jl

```diff
@@ -1,11 +1,11 @@
 using BinaryBuilder
 
 name = "licensecheck"
-version = v"0.3.1"
+version = v"000.300.101"
```

Comment:
Can do
```suggestion
version = v"0.3.1"
```
still since @giordano says the build version will get bumped anyway